client/shaders/3d_interlaced_merge/opengl_fragment.glsl

```diff
@@ -1,3 +1,5 @@
+#line 1
```

Comment:
You know that shader.cpp could automatically prepend this, right?

Reply:
Kind of realized afterwards :)

Reply:
It's the proper way now.